File: tests/benchmarks/fizzbuzz.js
for (let counter = 1; counter <= 10000; counter++) {
	let result = "";

	if (counter % 3 == 0) {
		result += "fizz";
	}

	if (counter % 5 == 0) {
		result += "buzz";
	}

	if (result != "") {
		console.log(result);
	}
	else {
		console.log(counter);
	}
}

File: tests/benchmarks/fizzbuzz.lua
for counter = 1, 10000 do
	local result = ""

	if counter % 3 == 0 then
		result = result .. "fizz"
	end

	if counter % 5 == 0 then
		result = result .. "buzz"
	end

	if result ~= "" then
		print(result)
	else
		print(counter)
	end
end
